main.go

```diff
@@ -0,0 +1,124 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/microkit/command"
+	microserver "github.com/giantswarm/microkit/server"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/viper"
+
+	"github.com/giantswarm/app-operator/flag"
+	"github.com/giantswarm/app-operator/server"
+	"github.com/giantswarm/app-operator/service"
+)
+
+const (
+	notAvailable = "n/a"
+)
+
+var (
+	description = "The app-operator deploys catalogs' app by look up to appCatalog CR."
+	f           = flag.New()
+	name        = "app-operator"
+	gitCommit   = notAvailable
+	source      = "https://github.com/giantswarm/app-operator"
+)
+
+func main() {
+	err := mainWithError()
+	if err != nil {
+		panic(fmt.Sprintf("%#v\n", err))
+	}
+}
+
+func mainWithError() (err error) {
+	// Create a new logger that is used by all packages.
+	var newLogger micrologger.Logger
+	{
+		c := micrologger.Config{
+			IOWriter: os.Stdout,
```

Comment:
```suggestion
```

Reply:
The default is already the standard output descriptor. See https://github.com/giantswarm/micrologger/blob/master/default.go#L15.